db-scripts/src/main/resources/migration.sql

```diff
@@ -452,3 +453,29 @@ ALTER TABLE gistic_to_gene DROP FOREIGN KEY gistic_to_gene_ibfk_2;
 ALTER TABLE gistic_to_gene ADD CONSTRAINT `gistic_to_gene_ibfk_2` FOREIGN KEY (`GISTIC_ROI_ID`) REFERENCES `gistic` (`GISTIC_ROI_ID`) ON DELETE CASCADE;
 
 UPDATE info SET DB_SCHEMA_VERSION="2.6.0";
+
+##version: 2.7.0
+-- ================== new reference genome foreign key for mutation_event and genetic_profile tables   ==================
+ALTER TABLE `mutation_event` ADD COLUMN `REFERENCE_GENOME_ID` INT(4) NULL AFTER `ENTREZ_GENE_ID`;
+ALTER TABLE `mutation_event`
+    ADD FOREIGN KEY (`REFERENCE_GENOME_ID`) REFERENCES `reference_genome_gene` (`REFERENCE_GENOME_ID`) ON DELETE CASCADE;
+
+UPDATE `mutation_event` set REFERENCE_GENOME_ID = (SELECT `REFERENCE_GENOME_ID` FROM `reference_genome` WHERE BUILD_NAME = 'GRCh37') WHERE NCBI_BUILD in ('hg19', 'GRCh37','37');
+-- UPDATE `mutation_event` set REFERENCE_GENOME_ID = (SELECT `REFERENCE_GENOME_ID` FROM `reference_genome` WHERE BUILD_NAME = 'GRCm38') WHERE NCBI_BUILD in ('mm10', 'GRCm38');
```

Comment:
please enable this one as well, and include one for GRCh38 as well.

Reply:
@sandertan  `migrate_db.py` should run `select distinct(NCBI_BUILD) from mutation_event`. If this results in more than 1 build (e.g. 37 and 38), then the migration should give an error telling the user to start from seed DB (i.e. remove all studies first).